airbyte-integrations/connectors/destination-postgres-strict-encrypt/build.gradle

```diff
@@ -22,4 +22,7 @@ dependencies {
 
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
     integrationTestJavaImplementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)
+    integrationTestJavaImplementation project(':airbyte-config:config-models')
```

Comment:
Not a blocker, however can you enlighten me on the intent of these two lines?

Reply:
Test needed to reference these two project in order to be able to call the test method.
Otherwise compilation of `PostgresDestinationStrictEncryptAcceptanceTest` fails for not being able to import things like `WorkerException`. 
Ref was added only in the context of test: `integrationTestJavaImplementation`